.github/scripts/implement_agent.py

```
@@ -322,124 +325,89 @@ def _run_baseline_pytest() -> str:
-                model=model,
+            resp = client.messages.create(
+                model=OPUS_MODEL,
                 max_tokens=8192,
-                tools=TOOLS,  # type: ignore[arg-type]
-                tool_choice="auto",
-                messages=messages,  # type: ignore[arg-type]
+                system=SYSTEM,
+                tools=anthropic_tools,  # type: ignore[arg-type]
+                messages=messages,      # type: ignore[arg-type]
             )
         except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)